nodesharing-lib/src/main/java/com/redhat/jenkins/nodesharing/transport/CrumbResponse.java

```diff
@@ -49,4 +54,18 @@ public CrumbResponse(@Nonnull String crumb, @Nonnull String crumbRequestField) {
     public @Nonnull String getCrumbRequestField() {
         return crumbRequestField;
     }
+
+    public List<String> getCookies() {
```

Comment:
The mutability here violates the contract of `Entity` being immutable. Also, the `cookies` are not part of the response served. See above for recommended solution.